landing_page/Gemfile

```diff
@@ -50,9 +52,12 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rails-controller-testing'
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
+gem 'pry'
```

Comment:
These may be better put within either the `:development` or `:test` groups, as you wouldn't really need these in production.

Reply:
Resolved in commit 07d7e2f4dd05baad296d964d81b1d24900b8a7d0